### PR TITLE
Add Buxfer

### PR DESCRIPTION
**Pull request by ohheyrj**
Added buxfer.com, financial management platform.

2FA only supported on prime accounts (annoyingly) and software 2FA only by authy app (again, annoyingly!), these are noted in the exceptions.

Alexa ranking:
image

### CODE REVIEW

File: _data/finance.yml [hunk 1]
```diff
@@ -60,6 +60,15 @@ websites:
...
     doc: https://github.com/2factorauth/twofactorauth/pull/2181#issuecomment-266315173
 
+  - name: Buxfer
+    url: https://www.buxfer.com
+    img: buxfer.png
+    tfa:
+      - sms
+      - phone
+      - totp
+    exception: "Only supported on 'Prime' accounts, software 2FA via Authy app only."
```
Comment (phallobst) [inline _data/finance.yml]:
```suggestion
      - proprietary
    exception: "Only supported on 'Prime' accounts."
```

**Review by phallobst — CHANGES_REQUESTED** [review]
Is there any publicly accessible documentation for the 2FA support? Or could you provide screenshots (without personal information) that shows the setup process?

**Comment by ohheyrj** [comment]
> Is there any publicly accessible documentation for the 2FA support? Or could you provide screenshots (without personal information) that shows the setup process?

Hello! This is the only screenshot I can get at the moment as there seems to be an issue with me setting up 2FA.

Screenshot from 2020-06-07 21-35-30

**Comment by Carlgo11** [comment]
And you use this service yourself?

From looking at the [terms of use](https://www.buxfer.com/terms) the company seems to be, or borderline to, illegal. They take no responsibility for the security of their systems or data they collect; They don't disclose what kind of data that they collect or to which parties they share it with; No GDPR compliance whatsoever.

I'd recommend adding a 
```YAML
regions:
  - us
```
tag as they seem to only focus on the U.S.

**Comment by ohheyrj** [comment]
Yes, for my sins I do use this service.
I'm not sure about the tag, the API they use does have a large amount of international banks, probably the largest selection I've seen for the UK.

**Comment by Carlgo11** [comment]
It isn't legal in the UK though 😞

**Review by phallobst — APPROVED** [review]

**Comment by Carlgo11** [comment]
Closing due to inactivity.

**Pull request closed without merging**